examples/calib_pulses.py

```diff
@@ -0,0 +1,71 @@
+import pyspecdata as psd
+import os
+import SpinCore_pp as spc
+from datetime import datetime
+from Instruments import GDS_scope
+import numpy as np
+
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+# {{{ importing acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+# {{{ add file saving parameters to config dict
+config_dict["type"] = "pulse_calib"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["misc_counter"] += 1
+# }}}
+# {{{ ppg
+sqrt_P = config_dict["amplitude"] * np.sqrt(75)  # we have a 75 W amplifier
+prog_beta = np.linspace(10, 100, 50, endpoint=False)
```

Comment:
I want to acquire linearly spaced beta!

Reply:
units of us*sqrt(W)

Reply:
in general, please don't put comments in the PR that belong as code comments.

But also, this is not true.  You just want to acquire what the β value is for a set of linearly spaced pulse lengths.

You do want to use the amplitude to help you determine the longest pulse length that you're going to use -- using similar math to what you're using here, which would be smart.

You always want to start from something like 0.5 μs so that you can really see the nonlinearity that you're trying to calibrate for.

I do not understand why you're passing `endpoint=False` to linspace here.